frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/ViewYamlModal/helpers.tests.ts

```diff
@@ -0,0 +1,333 @@
+import { render, screen } from "@testing-library/react";
+import { createMockSoftwarePackage } from "__mocks__/softwareMock";
+import { noop } from "lodash";
+
+import {
+  hyphenatedSoftwareTitle,
+  createPackageYaml,
+  renderYamlHelperText,
+} from "./helpers";
+
+describe("hyphenatedSoftwareTitle", () => {
```

Comment:
good tests. would be good to have a case to try with numeric characters and symbols as well

Reply:
Ooo, I don't even know what the expected behavior for that is, thank you. Will talk to design and put in followup